kubetest2-gke/deployer/network_test.go

```diff
@@ -0,0 +1,103 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package deployer
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestPrivateClusterArgs(t *testing.T) {
+	testCases := []struct {
+		desc           string
+		network        string
+		accessLevel    string
+		masterIPRanges []string
+		clusterInfo    cluster
+		expected       []string
+	}{
+		{
+			desc:           "no private cluster args are needed for non-private clusters",
+			network:        "whatever-network",
+			accessLevel:    "",
+			masterIPRanges: []string{"whatever-master-IP-range"},
```

Comment:
nit: for non private clusters, we need a (new/modify this) case where masterIPRanges could be empty.

Reply:
Added the test case, thanks